cypress/e2e/14-data-transformation-expressions.cy.ts

```diff
@@ -12,11 +12,12 @@ describe('Data transformation expressions', () => {
 	beforeEach(() => {
 		wf.actions.visit();
 
-		cy.window()
-			// @ts-ignore
-			.then(
-				(win) => win.onBeforeUnloadNodeView && win.removeEventListener('beforeunload', win.onBeforeUnloadNodeView),
-			);
+		cy.window().then(
+			(win) => {
+				// @ts-ignore
```

Comment:
could we augment the typeof `win` to have `preventNodeViewBeforeUnload?: boolean`, so we don't need to sprinkle `// @ts-ignore` in so many places ?